packages/node_modules/@webex/plugin-meetings/test/unit/spec/space-meeting.js

```diff
@@ -0,0 +1,94 @@
+
+import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
+
+import testUtils from './testUtils';
+
+require('dotenv').config();
+
+const sparkTestUsers = require('./sparkTestUsers.js');
+
+let userSet, alice, bob, chris;
+
+xdescribe('plugin-meeting-space-meeting', () => {
```

Comment:
?

Reply:
Will enable it in next phase for some reason it worked when i run it individually but doesn't when i run it together with 1:1

Reply:
The linter may reject this. I forget if it's a warning or error.